apps/src/code-studio/youtubeCheck.js

```diff
@@ -0,0 +1,14 @@
+export default function youTubeAvailabilityEndpointURL(noCookie) {
```

Comment:
Good call.  Updated filename to match function name.  

I've kept this in `apps/src/code-studio/` since we want it to be available to its original user in `apps/src/code-studio/videos.js` while also available for our player in `apps/src/standaloneVideo/`.  And I've kept `testYouTubeAvailable` in `apps/src/standaloneVideo/` since it's only used in our player.

Reply:
Sounds good!